extensions/wikia/Track/Track.php

```diff
@@ -100,6 +104,23 @@ public static function event ($event_type, $param=null) {
 	}
 
 	/**
+	 * Check whether events should be tracked
+	 *
+	 * @return bool
+	 */
+	protected static function shouldTrackEvents() {
+		// Check request headers to make sure this is not a mirrored
+		// request, which could result in duplicate tracking events
+		$headers = getallheaders();
```

Comment:
So this way works, but we already have a class which abstracts away getting the headers like this. What do you think about updating this method to use
```php
F::app()->wg->request->getHeader()
```
?